staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go

```diff
@@ -261,6 +283,27 @@ var (
 			},
 		},
 	)
+	SuggestedFlowSchemaSystemNodeHealth = newFlowSchema(
+		"system-nodes-health", "node-health", 400,
+		flowcontrol.FlowDistinguisherMethodByUserType,
+		flowcontrol.PolicyRulesWithSubjects{
+			Subjects: groups(user.NodesGroup), // the nodes group
+			ResourceRules: []flowcontrol.ResourcePolicyRule{
+				resourceRule(
+					[]string{flowcontrol.VerbAll},
+					[]string{corev1.GroupName},
+					[]string{"nodes", "nodes/status"},
+					[]string{flowcontrol.NamespaceEvery},
+					true),
+				resourceRule(
+					[]string{flowcontrol.VerbAll},
+					[]string{coordinationv1.GroupName},
+					[]string{"leases"},
```

Comment:
It's confusing to include leases in something named "node-health".

Maybe use "kubelet-high" instead?

Reply:
Naming is hard :)

I think nodes aren't just about Kubelet - IIRC, NodeProblemDetector is also running under "NodeGroup".
So I think kubelet is not the best.

I actually like node (but it's in a sense of "node" (as a machine/VM where we run pods) not in the context of "node" as an API object. But I see where the confusion is coming from.

I with I have a better proposal (node-heartbeats? [but that doesn't reflect the node status changes]).

Reply:
Oh, if it's NPD also, then how about "node-high"? (but should NPD really be included?)

Reply:
"node-high" sounds fine to me.

Re NPD - in my opinion it should, because if there is a problem with a node, I think reporting the status (to reflect the actual state and e.g. block scheduling more pods on that in case of issues) is more important than other operations (e.g. reporting new status of pod or fetching secrets to start it or more like that).

Reply:
Thanks for naming suggestions :) I renamed to "node-high" :)